### PR TITLE
CIV-4233 - add controller and views modifications

### PR DESCRIPTION
**Pull request by laarmada**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-4233

### Change description ###

'Enter your details' screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -375,28 +375,30 @@
       "SAVINGS_ACCOUNT": "Cyfrif cynilo"
     },
     "CITIZEN_DETAILS": {
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/modules/i18n/locales/cy.json]: Thanks Maro, in order to align the translation files, I override the translation. Good point.

**Review by portizdev — APPROVED** [review]

**Review by commitBlob — APPROVED** [review]

File: src/main/routes/features/claim/yourDetails/claimantDetailsController.ts [hunk 2]
```diff
@@ -16,58 +18,77 @@ import {
...
-    renderPage(res, req, party, claimantIndividualAddress, claimantIndividualCorrespondenceAddress, claimantType);
+    renderPage(res, req, party, claimantIndividualAddress, claimantIndividualCorrespondenceAddress, claimantDetails, partyType);
   } catch (error) {
     next(error);
   }
 });
 
-claimantOrganisationDetailsController.post([CLAIMANT_ORGANISATION_DETAILS_URL, CLAIMANT_COMPANY_DETAILS_URL], async (req: AppRequest | Request, res: Response, next: NextFunction) => {
+claimantDetailsController.post(detailsURLs, async (req: AppRequest | Request, res: Response, next: NextFunction) => {
+  const partyType = getPartyTypeDependingOnRoute(req?.url);
```
Comment (GorilaNaranja) [inline src/main/routes/features/claim/yourDetails/claimantDetailsController.ts:72]: Same here

Comment (laarmada) [reply src/main/routes/features/claim/yourDetails/claimantDetailsController.ts]: ok

File: src/main/routes/features/claim/yourDetails/claimantDetailsController.ts [hunk 3]
```diff
@@ -16,58 +18,77 @@ import {
...
+      claimantDetails,
+      type: partyType,
+    });
+  }
 }
 
-claimantOrganisationDetailsController.get([CLAIMANT_ORGANISATION_DETAILS_URL, CLAIMANT_COMPANY_DETAILS_URL], async (req: AppRequest, res: Response, next: NextFunction) => {
+claimantDetailsController.get(detailsURLs, async (req: AppRequest, res: Response, next: NextFunction) => {
   try {
+    const partyType = getPartyTypeDependingOnRoute(req?.url);
```
[the remark this answers is not in the document]
Comment (laarmada) [reply src/main/routes/features/claim/yourDetails/claimantDetailsController.ts]: ok

**Pull request merged**